src/main/java/com/google/sps/data/friend_map/UserNode.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.friend_map;
+
+import java.util.Collection;
+import com.google.common.collect.ImmutableSet;
+
+/**
+* Class to represent a User in the map representing all user friendships
+*/
+public class UserNode {
+
+  private final String userID;
+  private ImmutableSet<String> currentUserFriendIDs;
+
+  /**
+  * Constructs an instance of a UserNode
+  *
+  * @param userID The unique id number of each user (obtained from Facebook API)
+  * @param currentUserFriendIDs Immutable set of all of the IDs of a user's friends
+  */
+  public UserNode(String userID, Collection<String> currentUserFriendIDs) {
+    this.userID = userID;
+    this.currentUserFriendIDs = ImmutableSet.copyOf(currentUserFriendIDs);
+  }
+
+  public String getUserID() {
+    return this.userID;
+  }
+
+  public ImmutableSet<String> getCurrentUserFriendIDs() {
```

Comment:
This class is essentially immutable (you have no setters). Do we need to say current? We certainly don't need user here. To me, this can just be `getFriendsIds()`

Reply:
Got it! I'll also go ahead and change the field name to friendIDs